cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -157,6 +157,21 @@ func (cm *certManagerClient) EnsureWebhook() error {
 	return nil
 }
 
+func (cm *certManagerClient) getWaitTimeout() time.Duration {
+	log := logf.Log
+
+	timeout, err := cm.configClient.CertManagerTimeout().Get()
```

Comment:
Can we do `o.configClient.Variables.Get(overrideFolderKey)` like we are already doing for e.g. `overrideFolders`
https://github.com/kubernetes-sigs/cluster-api/blob/d5b9a37cc4efdf4872d266e3221d495e338a483b/cmd/clusterctl/client/repository/overrides.go#L69

Reply:
I agree with leveraging the Variables Client. 
As an example if you define  `certManagerTimeout` similar to the `overridesFolder` in clusterctl.yaml. It should be made accessible via the variables client. Then we don't need ot have a specific client just for a component's timeout.

https://github.com/kubernetes-sigs/cluster-api/blob/09f87a2aadcd590952e17684f5c05259337e4da9/cmd/clusterctl/test/run-e2e.sh#L61

Reply:
ok, I was just trying to follow the pattern I found, will re-work it